src/state/apis/snapshot/snapshot.ts

```diff
@@ -52,8 +77,13 @@ export const snapshotApi = createApi({
       queryFn: async (_, { dispatch, getState }) => {
         const accountIds: AccountId[] =
           (getState() as ReduxState).portfolio.accountMetadata.ids ?? []
-        const strategiesResult = await dispatch(snapshotApi.endpoints.getStrategies.initiate())
-        const strategies = strategiesResult?.data
+        const strategies = await (async () => {
+          const maybeSliceStragies = (getState() as ReduxState).snapshot.strategies
+          if (maybeSliceStragies) return maybeSliceStragies
```

Comment:
are we just assuming strategies won't change? or where is this invalidated/refetched ever?

Reply:
Yeah strategies are assumed to never change - realistically, this could even be a constant, but might as well keep it for when they do so we can just do a migration and get fresh ones

Reply:
perfect, makes sense and thanks for the clarification!